### PR TITLE
776: Add Maltese to locales.php

### LINKED ISSUE #776: Add support for Maltese
The PR happened to be submitted before the corresponding issue was created. (Corresponding PR: https://github.com/GlotPress/GlotPress-WP/pull/761 )
At https://make.wordpress.org/polyglots/2017/06/27/hello-10/ we've received a request to add support for Maltese.

Ethnologue ref: https://www.ethnologue.com/language/mlt

Data:
```
		$mlt = new GP_Locale(); 
		$mlt->english_name = 'Maltese';
		$mlt->native_name = 'Malti';
		$mlt->lang_code_iso_639_1 = 'mt';
		$mlt->lang_code_iso_639_2 = 'mlt';
		$mlt->lang_code_iso_639_3 = 'mlt';
		$mlt->country_code = 'mt';
		$mlt->facebook_locale = 'mt_MT';
		$mlt->google_code = 'mt';
		$mlt->wp_locale = 'mlt';
		$mlt->slug = 'mlt';
		$mlt->nplurals = 4;
		$mlt->plural_expression = '(n==1 ? 0 : n==0 || ( n%100&gt;1 && n%10010 && n%100&lt;20 ) ? 2 : 3);';

```

### PR DESCRIPTION
**Pull request by tobifjellner**
Adds support for Maltese in GlotPress. Fixes #776

### CODE REVIEW

**Comment by SergeyBiryukov** [comment]
Related request: https://make.wordpress.org/polyglots/2017/06/27/hello-10/
Related Ethnologue information: https://www.ethnologue.com/language/mlt

File: locales/locales.php [hunk 1]
```diff
@@ -1388,6 +1388,18 @@ public function __construct() {
...
+		$mt->english_name = 'Maltese';
+		$mt->native_name = 'Malti';
+		$mt->lang_code_iso_639_1 = 'mt';
+		$mt->lang_code_iso_639_2 = 'mlt';
+		$mt->lang_code_iso_639_3 = 'mlt';
+		$mt->country_code = 'mt';
+		$mt->wp_locale = 'mlt';
+		$mt->slug = 'mt';
+		$mt->nplurals = 4;
+		$mt->plural_expression = '(n==1 ? 0 : n==0 || ( n%100&gt;1 &amp;&amp; n%10010 &amp;&amp; n%100&lt;20 ) ? 2 : 3);';
```
Comment (ocean90) [inline locales/locales.php]: There's some escaped HTML.

File: locales/locales.php [hunk 2]
```diff
@@ -1388,6 +1388,18 @@ public function __construct() {
...
 
+		$mt = new GP_Locale(); 
+		$mt->english_name = 'Maltese';
+		$mt->native_name = 'Malti';
+		$mt->lang_code_iso_639_1 = 'mt';
+		$mt->lang_code_iso_639_2 = 'mlt';
+		$mt->lang_code_iso_639_3 = 'mlt';
+		$mt->country_code = 'mt';
+		$mt->wp_locale = 'mlt';
+		$mt->slug = 'mt';
```
Comment (ocean90) [inline locales/locales.php]: The slug should be `mlt`.

File: locales/locales.php [hunk 3]
```diff
@@ -1388,6 +1388,18 @@ public function __construct() {
 		$mg->google_code = 'mg';
 		$mg->facebook_locale = 'mg_MG';
 
+		$mt = new GP_Locale();
```
Comment (ocean90) [inline locales/locales.php]: The variable should be `$mlt`

File: locales/locales.php (hunk 2 above)
Comment (ocean90) [reply locales/locales.php]: The Google code is missing, it's `mt`.

**Review by ocean90 — CHANGES_REQUESTED** [review]
Thanks for the PR!

Please take a look at our [contributing guidelines](https://github.com/GlotPress/GlotPress-WP/blob/develop/.github/CONTRIBUTING.md#submitting-patches). They request that you first create an issue like https://github.com/GlotPress/GlotPress-WP/issues/411. Also, commit messages should include more than "Update xyz".

File: locales/locales.php (hunk 2 above)
Comment (ocean90) [reply locales/locales.php]: The Facebook code is missing, it's `mt_MT`.

**Review by ocean90 — CHANGES_REQUESTED** [review]

**Comment by tobifjellner** [comment]
Thanks for feedback. Will follow up soon.

File: locales/locales.php [hunk 4]
```diff
@@ -1388,6 +1388,20 @@ public function __construct() {
...
+		$mlt->lang_code_iso_639_1 = 'mt';
+		$mlt->lang_code_iso_639_2 = 'mlt';
+		$mlt->lang_code_iso_639_3 = 'mlt';
+		$mlt->country_code = 'mt';
+		$mlt->facebook_locale = 'mt_MT';
+		$mlt->google_code = 'mt';
+		$mlt->wp_locale = 'mlt';
+		$mlt->slug = 'mlt';
+		$mlt->nplurals = 4;
+		$mlt->plural_expression = '(n==1 ? 0 : n==0 || ( n%100&gt;1 && n%10010 && n%100&lt;20 ) ? 2 : 3);';
```
Comment (ocean90) [inline locales/locales.php]: No HTML here, `&gt;` needs to be `>`

Comment (ocean90) [reply locales/locales.php]: There's something missing at `n%10010`

File: locales/locales.php [hunk 5]
```diff
@@ -1388,6 +1388,20 @@ public function __construct() {
 		$mg->google_code = 'mg';
 		$mg->facebook_locale = 'mg_MG';
 
+		$mlt = new GP_Locale();
```
Comment (ocean90) [inline locales/locales.php]: Now that the key is `$mlt`, it needs to be moved after the definition of the `$ml` locale.

**Review by ocean90 — CHANGES_REQUESTED** [review]

**Comment by tobifjellner** [comment]
Thanks for your feedback. I believe it to be right now, but have also asked the original requester to confirm the plural forms at https://make.wordpress.org/polyglots/2017/06/27/hello-10/#comment-276325
By the way: Is it possible to rebase the diff via the webGUI of github?

File: locales/locales.php [hunk 6]
```diff
@@ -1419,6 +1419,20 @@ public function __construct() {
...
+		$mlt->lang_code_iso_639_1 = 'mt';
+		$mlt->lang_code_iso_639_2 = 'mlt';
+		$mlt->lang_code_iso_639_3 = 'mlt';
+		$mlt->country_code = 'mt';
+		$mlt->facebook_locale = 'mt_MT';
+		$mlt->google_code = 'mt';
+		$mlt->wp_locale = 'mlt';
+		$mlt->slug = 'mlt';
+		$mlt->nplurals = 4;
+		$mlt->plural_expression = '(n==1 ? 0 : n==0 || (n%100>=2 && n%100<=10) ? 1 : (n%100>=11 && n%100<=19) ? 2 : 3)';
```
Comment (ocean90) [inline locales/locales.php]: To make this consistent with other locales, please change `<=19` to `<20`.

File: locales/locales.php [hunk 7]
```diff
@@ -1419,6 +1419,20 @@ public function __construct() {
...
 
+		$mlt = new GP_Locale(); 
+		$mlt->english_name = 'Maltese';
+		$mlt->native_name = 'Malti';
+		$mlt->lang_code_iso_639_1 = 'mt';
+		$mlt->lang_code_iso_639_2 = 'mlt';
+		$mlt->lang_code_iso_639_3 = 'mlt';
+		$mlt->country_code = 'mt';
+		$mlt->facebook_locale = 'mt_MT';
+		$mlt->google_code = 'mt';
```
Comment (ocean90) [inline locales/locales.php]: To make this consistent with other locales, please move the Google and Facebook codes to the end of the locale definition.

**Review by ocean90 — CHANGES_REQUESTED** [review]

File: locales/locales.php [hunk 8]
```diff
@@ -1419,6 +1419,20 @@ public function __construct() {
...
+		$mlt->lang_code_iso_639_1 = 'mt';
+		$mlt->lang_code_iso_639_2 = 'mlt';
+		$mlt->lang_code_iso_639_3 = 'mlt';
+		$mlt->country_code = 'mt';
+		$mlt->wp_locale = 'mlt';
+		$mlt->slug = 'mlt';
+		$mlt->nplurals = 4;
+		$mlt->plural_expression = '(n==1 ? 0 : n==0 || ( n%100>1 && n%100<11) ? 1 : (n%100>10 && n%100<20 ) ? 2 : 3)';
+		$mlt->facebook_locale = 'mt_MT';
+		$mlt->google_code = 'mt';
```
Comment (ocean90) [inline locales/locales.php]: Last request: Google should be before Facebook. 🙈

**Review by ocean90 — CHANGES_REQUESTED** [review]

**Review by ocean90 — APPROVED** [review]

**Pull request merged**